src/commands/support/airac.ts

```diff
@@ -0,0 +1,29 @@
+import { CommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+import { makeEmbed, makeLines } from '../../lib/embed';
+
+export const airac: CommandDefinition = {
+    name: 'airac',
+    description: 'Provides information about free SimBrief account AIRAC limitations',
+    category: CommandCategory.SUPPORT,
+    executor: async (msg) => {
+        const airacEmbed = makeEmbed({
+            title: 'FlyByWire Support | SimBrief AIRACs',
+            description: makeLines ([
+                'Free SimBrief accounts are limited to generating routes valid for AIRAC 2003, while MSFS currently uses AIRAC 2203. '
```

Comment:
```suggestion
                'Free SimBrief accounts are limited to generating routes valid for AIRAC 2003, while MSFS regularly updates to the latest AIRAC available. '
```

Makes this command more future proof against AIRAC changes.

Reply:
This is an especially good suggestion for futureproofing